gremlin-javascript/src/main/javascript/gremlin-javascript/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "gremlin-javascript",
+  "version": "3.2.7-alpha1",
```

Comment:
should be 3.2.8-alpha1 - at this point. though, we've been using "release candidate" suffix for other GLVs as in "3.2.8-rc1" - perhaps we should continue in that vein?

Reply:
I've used `alpha` to avoid messing up in case it was accidentally pushed while testing the pull request.
The version suffix is added in the `determineVersion()` method of the `generation.groovy` file.